S23/kishore223/lowercase.py

```diff
@@ -0,0 +1,24 @@
+def lowercase_word(word, lang):
+    if lang in ['tr', 'az']:
+        return word.replace('I', 'ı').lower()
+    elif lang == 'ga':
+        vowels = set(['A', 'E', 'I', 'O', 'U', 'Á', 'É', 'Í', 'Ó', 'Ú'])
+        for i in range(1, len(word)):
+            if word[i-1] in ['n', 't'] and word[i] in vowels and word[i].isupper():
+                return word[:i] + '-' + word[i:].lower()
+        return word.lower()
+    elif lang == 'el':
+        if word.endswith('Σ'):
+            return word[:-1] + 'ς'
+        else:
+            return word.lower()
+    elif lang in ['zh', 'ja', 'th']:
+        return word
+    else:
+        return word.lower()
+
+word = input("Enter a word: ")
+lang = input("Enter language code (BCP-47): ")
+
+lowercased_word = lowercase_word(word, lang)
```

Comment:
if some person will enter any random BCP-47 code which is not existing then it will convert text into lower case but after the output coming how person can know that the language code is correct or not so should should we ask for language inspite of language code?

Reply:
Now I have mentioned the code word for every language.